packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -1142,29 +1143,29 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         >
           {' '}
           {
-            <span ref={isSelected ? this._selectedElement : undefined}>
+            <span className={optionClassNames.optionTextWrapper} ref={isSelected ? this._selectedElement : undefined}>
```

Comment:
is it possible to move this inside the _onRenderOptionContent function? This way it won't affect anyone using a custom render function.

Reply:
That's what I'm trying to figure out right now. After I sent out the PR, it seems like that's what we're trying to do in the CSS, but it's not happening.

Reply:
is this something you can repro in a codepen easily? overflow gets tricky with nested elements

Reply:
I'm not sure about how easily it is to repro, however after playing around with it, I was able to change the css so that the only CSS that we add to optionTextWrapper is setting the maxWidth to be 100% of it's container, which I imagine is a safe change since our items should already be 100% of the container anyways.

By doing this, all we need to do now is set a width/maxWidth to the whole container (optionsContainerWrapper) and now we can constrain the size of the combo box option to a set size of our choice.

Reply:
@micahgodbolt Currently we can set a size for the combo box to be able to constrain our options to a certain size, but we don't use it anywhere.

Is there any fix sizes we want to use? Or are we okay with just leaving it unset and let our combobox be as large as the option's text?

Reply:
Okay, I'm going to use that information to decide the size of the ComboBox.

Reply:
Isn't that already happening?

Reply:
I dug a bit into it, it's being set from the callout and not the getStyles for the combobox.styles which allows us to pass in a width, which we never use.